### PR TITLE
Show vote delegation in `stake-address-info`

### LINKED ISSUE #423: stake-address-info to show drep-delegation
Currently, the stake-address-info shows:

```
cardano-cli conway query stake-address-info --testnet-magic 42 --address stake_test1uqjmy360gwmmdh02jeez2xunuanu35vvxzg94rdqa44svdsg3wuvt
[
    {
        "address": "stake_test1uqjmy360gwmmdh02jeez2xunuanu35vvxzg94rdqa44svdsg3wuvt",
        "delegation": "pool16nfsdtcszpfquqx038q7uz9ycvktnkr5r8xs6vs27uslqj3yw5k",
        "delegationDeposit": 0,
        "rewardAccountBalance": 0
    }
]%
```
It would be stellar to also include drep delegation on the output. Something like 

```
cardano-cli conway query stake-address-info --testnet-magic 42 --address stake_test1uqjmy360gwmmdh02jeez2xunuanu35vvxzg94rdqa44svdsg3wuvt
[
    {
        "address": "stake_test1uqjmy360gwmmdh02jeez2xunuanu35vvxzg94rdqa44svdsg3wuvt",
        "stakeDelegation": "pool16nfsdtcszpfquqx038q7uz9ycvktnkr5r8xs6vs27uslqj3yw5k",
        "voteDelegation": "cf09b59e134fa14e48da39b552c02299a054d7c8b895b3d827453672",
        "delegationDeposit": 0,
        "rewardAccountBalance": 0
    }
]%
```
```
cardano-cli conway query stake-address-info --testnet-magic 42 --address stake_test1uqjmy360gwmmdh02jeez2xunuanu35vvxzg94rdqa44svdsg3wuvt
[
    {
        "address": "stake_test1uqjmy360gwmmdh02jeez2xunuanu35vvxzg94rdqa44svdsg3wuvt",
        "stakeDelegation": "pool16nfsdtcszpfquqx038q7uz9ycvktnkr5r8xs6vs27uslqj3yw5k",
        "voteDelegation": "alwaysAbstain",
        "delegationDeposit": 0,
        "rewardAccountBalance": 0
    }
]%
```

### PR DESCRIPTION
**Pull request by carlhammann**
# Changelog

```yaml
- description: |
    add the vote delegateee to the output of `stake-address-info`
# uncomment types applicable to the change:
  type:
  # - feature        # introduces a new feature
  # - breaking       # the API has changed in a breaking way
  - compatible     # the API has changed but is non-breaking
  # - optimisation   # measurable performance improvements
  # - improvement    # QoL changes e.g. refactoring
  # - bugfix         # fixes a defect
  # - test           # fixes/modifies tests
  # - maintenance    # not directly related to the code
  # - release        # related to a new release preparation
  # - documentation  # change in code docs, haddocks...
```

# Context

This PR closes #423.

# How to trust this PR

Highlight important bits of the PR that will make the review faster. If there are commands the reviewer can run to observe the new behavior, describe them.

# Checklist

- [ ] Commit sequence broadly makes sense and commits have useful messages
- [ ] New tests are added if needed and existing tests are updated. See [Running tests](https://github.com/input-output-hk/cardano-node-wiki/wiki/Running-tests) for more details
- [ ] Self-reviewed the diff

### CODE REVIEW

**Comment by carlhammann** [comment]
With last Friday's integration work on the node, this should now be usable with current main `cardano-node`. I'm therefore opening this for review. I'll release a new version of the `cardano-api` and revert [77bbc9d](https://github.com/input-output-hk/cardano-cli/pull/452/commits/77bbc9d61c8402f1a777cc37118bbb134c2eb9ce) before merging; that should also solve the CI failures.

File: cardano-cli/src/Cardano/CLI/EraBased/Run/Query.hs [hunk 1]
```diff
@@ -826,37 +833,48 @@ writeStakeAddressInfo
...
   jsonInfo :: [Aeson.Value]
   jsonInfo =
     map
-      (\(addr, mBalance, mPoolId, mDeposit) ->
+      (\(addr, mBalance, mPoolId, mDRep, mDeposit) ->
         Aeson.object
         [ "address" .= addr
-        , "delegation" .= mPoolId
+        , "stakeDelegation" .= mPoolId
+        , "voteDelegation" .= fmap friendlyDRep mDRep
```
Comment (carbolymer) [inline cardano-cli/src/Cardano/CLI/EraBased/Run/Query.hs:854]: can you add a golden test for this command output please?

**Review by carbolymer — APPROVED** [review]

File: cardano-cli/src/Cardano/CLI/EraBased/Run/Query.hs (hunk 1 above)
Comment (carlhammann) [reply cardano-cli/src/Cardano/CLI/EraBased/Run/Query.hs:854]: Running this command requires a connection to a live node. We haven't currently got tests for any such commands. That's what `cardano-testnet` will be for, once it's there.

Comment (carbolymer) [reply cardano-cli/src/Cardano/CLI/EraBased/Run/Query.hs:854]: Ah right. I don't know why I thought this was an offline command. This should be a part of a separate change to`cardano-testnet` then.

**Pull request merged**